uber/custom_tags.py

```diff
@@ -197,22 +197,28 @@ def remove_newlines(string):
     return string.replace('\n', ' ')
 
 
+form_link_site_sections = {}
```

Comment:
What's the purpose of this dictionary? I see that we reference it on line 215, but I can't figure out what it actually does or where it's populated.

Reply:
This dictionary exists so other plugins can add custom paths to the `form_link()` method. See here: https://github.com/magfest/panels/blob/b78e1105d1fd6c59e6c1fa07b8d7be5714a09324/panels/config.py#L100

It's kinda janky, but it fulfilled an immediate need. You may have noticed, as Super MAGFest approaches, my code is getting less and less disciplined 😓